kumo-core/src/test/java/com/kennycason/kumo/examples/DataRankWordCloudITest.java

```diff
@@ -22,6 +25,8 @@
 /**
  * Created by kenny on 2/24/16.
  */
+@Category(IntegrationTest.class)
```

Comment:
I'll figure out how to clean this up later. I'm not sure what my original intent was. :)